config/storage.yml

```diff
@@ -1,2 +1,36 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
```

Comment:
I think the above needs to be deleted  as well

Reply:
deleted!